src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -377,7 +378,7 @@ const supplierRequisitionInitialiser = requisition => {
     modalKey: '',
     hasSelection: false,
     modalValue: null,
-    showAll: !usingPrograms,
+    showAll: usingPrograms && isFinalised,
```

Comment:
I guess this is logical, given we ignore this field when `usingPrograms === false` anyway, but it seems to make more sense in my head to read it as this being true when NOT using programs OR is finalised .. maybe I am just crazy

Reply:
You're not crazy - at least regarding this topic ;-) - . Now I can see what you mean and makes sense to me as well. `!usingPrograms || isFinalised` is totally explicit and much clearer than the one I was using before. Working on this right now!